### PR TITLE
feat: explorer link on tx detail screen

### LINKED ISSUE #78: Add link to explorer on Transaction Detail screen

### PR DESCRIPTION
**Pull request by pedroferreira1**
Closes #78 

image

### CODE REVIEW

File: src/screens/CreateToken.js [hunk 1]
```diff
@@ -187,6 +192,25 @@ class CreateToken extends React.Component {
...
   }
 
+  /**
+   * Method called when user clicked to see the token deposit RFC
+   *
+   * @param {Object} e Event for the click
+   */
+  goToRFC = (e) => {
+    e.preventDefault();
+    const url = "https://gitlab.com/HathorNetwork/rfcs/blob/master/text/0011-token-deposit.md";
```
Comment (msbrogli) [inline src/screens/CreateToken.js]: I think we should move it to the constants file.

File: src/screens/TransactionDetail.js [hunk 2]
```diff
@@ -92,16 +97,53 @@ class TransactionDetail extends React.Component {
...
   }
 
+  /**
+   * Method called when user clicked on 'See on explorer' link
+   *
+   * @param {Object} e Event for the click
+   */
+  goToExplorer = (e) => {
+    e.preventDefault();
+    const url = `https://explorer.hathor.network/transaction/${this.state.transaction.hash}`;
```
Comment (msbrogli) [inline src/screens/TransactionDetail.js]: I think the `BASE_URL` should be in the constants file.

Comment (obiyankenobi) [reply src/screens/TransactionDetail.js]: I'd call it `EXPLORER_BASE_URL` or something like that so it's clear this refers to the explorer.

Comment (msbrogli) [reply src/screens/TransactionDetail.js]: @obiyankenobi I agree.

File: src/screens/CreateToken.js (hunk 1 above)
Comment (pedroferreira1) [reply src/screens/CreateToken.js]: Done

File: src/screens/TransactionDetail.js (hunk 2 above)
Comment (pedroferreira1) [reply src/screens/TransactionDetail.js]: Done

File: src/screens/CreateToken.js [hunk 3]
```diff
@@ -215,7 +228,7 @@ class CreateToken extends React.Component {
         <p className="mt-5">{t`Here you will create a new customized token. After the creation, you will be able to send this new token to other addresses.`}</p>
         <p>{t`Custom tokens share the address space with all other tokens, including HTR. This means that you can send and receive tokens using any valid address.`}</p>
         <p>{t`Remember to make a backup of your new token's configuration string. You will need to send it to other people to allow them to use your new token.`}</p>
-        <p>When creating and minting tokens, a <strong>deposit of {hathorLib.tokens.getDepositPercentage() * 100}%</strong> in HTR is required. If these tokens are later melted, this HTR deposit will be returned. Read more about it <a target="_blank" rel="noopener noreferrer" href="https://gitlab.com/HathorNetwork/rfcs/blob/master/text/0011-token-deposit.md">here</a>.</p>
+        <p>When creating and minting tokens, a <strong>deposit of {hathorLib.tokens.getDepositPercentage() * 100}%</strong> in HTR is required. If these tokens are later melted, this HTR deposit will be returned. Read more about it <a href="true" onClick={this.goToRFC}>here</a>.</p>
```
Comment (msbrogli) [inline src/screens/CreateToken.js]: Missing i18n markers.

Comment (pedroferreira1) [reply src/screens/CreateToken.js]: I will create another PR to mark this string. It was already not marked.

**Pull request merged**